pkg/eval/builtin_fn_cmd.go

```diff
@@ -118,7 +122,19 @@ func exit(fm *Frame, codes ...int) error {
 	panic("os.Exit returned")
 }
 
+// decSHLVL decrements $E:SHLVL. It's primary purpose is to ensure that
```

Comment:
Two nitpicks:

* Doc comment for unexported symbols: https://github.com/elves/elvish/blob/master/CONTRIBUTING.md#comment-for-unexported-go-types-and-functions. This is new, BTW, and there is probably some existing code that doesn't follow this.

* It's -> Its